File: models/objects/plans_.js
var mongoose = require('mongoose'),
	Schema = mongoose.Schema,
	_ = require('underscore'),
	returnJSON = require('./utils').returnJSON;

var plans_schema = new Schema({
 	name: String,
	price: String,
	parrots: Number,
	id: String,
	product_url: String,
	product_path: String,
	created_at: {type: Date, default: Date.now, private: true},
});

mongoose.model('Plans', plans_schema);
var Plans = mongoose.model('Plans');
module.exports = Plans;

Plans.prototype.returnJSON = returnJSON;

File: dal/payparrot_dal/plans.py
from datetime import datetime
from hashlib import sha256
from random import random
from time import time

from payparrot_dal.base import BaseModel

class Plans(BaseModel):
    _meta = {
        'collection': 'plans',
        'fields': {
            'name': {'required': True},
            'price': {},
            'parrots': {},
            'product_url': {},
            'product_path': {},
            'created_at': {'private': True, 'default': datetime.now}
        }
    }